<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorDOF</class>
 <widget class="QWidget" name="InspectorDOF">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Depth of Field (DOF)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="1">
         <widget class="QLineEdit" name="m_pNearToFarRatioEdit"/>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="m_pFadeUnSmallEdit"/>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="m_pFarEdit"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Fade Small- to medium blur</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="m_pFadeSmallMediumEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Far Distance</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="m_pNearEdit"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Near Distance </string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Fade Un- to Small blur</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Near to far ratio</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_pNearEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>InspectorDOF</receiver>
   <slot>valueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pFarEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>InspectorDOF</receiver>
   <slot>valueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pNearToFarRatioEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>InspectorDOF</receiver>
   <slot>valueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pFadeUnSmallEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>InspectorDOF</receiver>
   <slot>valueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pFadeSmallMediumEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>InspectorDOF</receiver>
   <slot>valueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>valueChanged()</slot>
 </slots>
</ui>
